### PR TITLE
Updating node week 3 to address Westmidlands class feedback

### PR DESCRIPTION
**Pull request by sansaid**
## What does this change?

Module: Node
Week(s): 3

## Description

After completing this session with the Westmidlands class 3, we found the following problems (resulting changes to the codebase are below the problem) - appreciate these could be very specific solutions to WM, so opened the PR to start the discussion about how to address these!

1. Not all students had access to their albums project for various reasons (changing PCs, some students are just really behind, etc.)
    a. Introducing a Glitch project with some albums that the students can modify 
2. The Heroku guide assumes too much has been setup on the students' private repo for them to publish the app using just the guide
    a. For students publishing their albums project, a video guide (that has been tested by one of the TAs) has been included. Students without the albums project are encouraged to follow the Heroku guide
3. Some students struggled to understand what some of the commands in the Heroku guide meant
    a. Some command breakdowns are provided for the two commands that students complained about the most 
4. The students felt overwhelmed by their first, full stack pair programming exercise
    a. We agreed we shouldn't change this and let the students "struggle together" on their first real development/collab project. We instead compromised by making the other 2 exercises in the lesson "completeable". We did, however, agree we should change the wording in the syllabus to remind students that they're not expecting to complete the project and aim to complete level 100 instead. We also added a tip to remind students to use the `create-react-app` command.

## Who needs to know about this?

@textbook @ChrisOwen101 @SallyMcGrath 

## Rendered Pages

-----
[View rendered docs/node/week-3/lesson.md](https://github.com/sansaid/syllabus/blob/node-week-3-refinements/docs/node/week-3/lesson.md)

### CODE REVIEW

**Review by illicitonion — APPROVED** [review]
Looks great - thanks so much!

**Review by SallyMcGrath — APPROVED** [review]
LGTM 🎉

File: docs/node/week-3/lesson.md [hunk 1]
```diff
@@ -106,31 +121,38 @@ You should complete all of the following sections:
...
 
 ## 4) City Guide and Paired Programming
 
 :::note Exercise
 
-For this exercises you should split into groups of two or three people.
+For this exercises you should split into groups of two or three people. Focus on just completing level 100 if you can! This exercise is designed to test your pair programming skills and to get you thinking about how to design a full stack application! :) You're not expected to get everything completed.
 
-This challenge should be completed in a Pair Programming style.
+You can either choose to split your focus (such that one person focuses on React and the other focuses on Node) or "swarm" on each side (i.e. both of you working on React or Node at one time - recommendation is to start on the side that at least one of you are not comfortable with!)
```
Comment (illicitonion) [inline docs/node/week-3/lesson.md]: From our chat in-meeting: Let's just suggest swarming, and explicitly tell them _not_ to assign a frontend person and a backend person.

**Comment by SallyMcGrath** [comment]
@all-contributors please add @sansaid for content and code

**Pull request merged**